apps/admin/frontend/package.json

```diff
@@ -153,25 +152,11 @@
     "@types/testing-library__jest-dom": "^5.14.3",
     "@types/uuid": "^8.3.4",
     "@types/zip-stream": "workspace:*",
-    "@typescript-eslint/eslint-plugin": "5.37.0",
-    "@typescript-eslint/parser": "5.37.0",
     "@vitejs/plugin-react": "^1.3.2",
     "@votingworks/admin-backend": "workspace:*",
     "@votingworks/grout-test-utils": "workspace:*",
     "@votingworks/monorepo-utils": "workspace:*",
     "@votingworks/test-utils": "workspace:*",
-    "eslint": "8.23.1",
-    "eslint-config-airbnb": "^19.0.4",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-config-react-app": "^7.0.1",
-    "eslint-import-resolver-node": "^0.3.4",
-    "eslint-plugin-cypress": "^2.12.1",
-    "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-jest": "^27.2.3",
-    "eslint-plugin-jsx-a11y": "^6.6.1",
-    "eslint-plugin-react": "^7.31.8",
-    "eslint-plugin-react-hooks": "^4.6.0",
```

Comment:
`react-hook`s was no longer necessary because the rules are bundled with `airbnb`